server/src/main/java/org/jboss/as/server/controller/resources/ServerRootResourceDefinition.java

```diff
@@ -444,6 +449,9 @@ public String getProductName() {
         // Paths
         resourceRegistration.registerSubModel(PathResourceDefinition.createSpecified(pathManager));
 
+        //capability registry
+        resourceRegistration.registerSubModel(new CapabilityRegistryResourceDefinition(capabilityRegistry));
+
```

Comment:
I *may* want to disable this before CR1 just because there's no client that will use it yet, and committing to support the end-user API for 10 years is a big step to take if we don't have to. What you have seems ok, so this is really just me being cautious. I'm a bit iffy about the point.getAddress().toCLIStyleString() for the reg points. We do that elsewhere but this data may be more directly consumed by the console, while other spots we do that (I think some RBAC config data) the console at most would just present the info.